app/assets/javascripts/Components/test_run_table.jsx

```diff
@@ -150,39 +150,35 @@ class TestGroupResultTable extends React.Component {
           return rows[0]['test_group_name'];
         }
       },
-    },
-    {
-      id: 'output',
-      Header: I18n.t('activerecord.attributes.test_result.output'),
-      accessor: row => row['test_results.output'] || '',
-      Cell: ({value}) => <pre className={'test-results-output'}>{value}</pre>,
-      show: this.state.show_output,
-      aggregate: _ => '',
-      className: 'actual_output'
+      minWidth: 200
     },
     {
       id: 'test_status',
       Header: I18n.t('activerecord.attributes.test_result.status'),
       accessor: row => row['test_results.status'],
-      minWidth: 50,
+      width: 80,
       aggregate: _ => ''
     },
     {
       id: 'marks_earned',
       Header: I18n.t('activerecord.attributes.test_result.marks_earned'),
       accessor: row => row['test_results.marks_earned'],
-      minWidth: 40,
+      Cell: row => {
+        const marksEarned = row.original['test_results.marks_earned'];
+        const marksTotal = row.original['test_results.marks_total'];
+        return `${marksEarned} / ${marksTotal}`;
```

Comment:
marksEarned or marksTotal can be null if the result hasn't returned yet. I would recommend replacing this with:

```js
if (marksEarned !== null && marksTotal !== null) {
   return `${marksEarned} / ${marksTotal}`;
}
return '';
```